packit_service/service/api/webhooks.py

```diff
@@ -219,25 +219,25 @@ def create_confidential_issue_with_token(self):
             namespace=namespace, repo_name=repo_name, project_url=http_url
         )
 
-        if not project_authentication_issue.issue_created:
+        if not project_authentication_issue:
             token = jwt.encode(
                 {"namespace": namespace, "repo_name": repo_name},
                 config.gitlab_token_secret,
                 algorithm="HS256",
-            )
+            ).decode("utf-8")
 
             project = config.get_project(url=http_url)
             packit_user = project.service.user.get_username()
 
             project.create_issue(
                 title="Packit-Service Authentication",
                 body=f"To configure Packit-Service with `{repo_name}` you need to\n"
-                f"configure the webhook settings. Head to {http_url}/hooks and add\n"
+                f"configure the webhook settings. Head to {http_url[:-4]}/hooks and add\n"
```

Comment:
You can use [project.get_web_url method](https://github.com/packit/ogr/blob/176add79eeb7d71e765550da76c9cdc8aced9e92/ogr/services/gitlab/project.py#L554) instead for better readability:

```suggestion
                f"configure the webhook settings. Head to {project.get_web_url()}/hooks and add\n"
```